fuzzing/libfuzzer/libs/gramine_fuzzing_libs.py

```diff
@@ -0,0 +1,73 @@
+import os
+from libs import utils
+from common.config.constants import *
+import subprocess
+
+
+def build_libfuzzer(filesize):
+    with open(os.path.join(LIBFUZZER_CORPUS_DIR, str(filesize)), 'wb') as file:
+        file.write(os.urandom(filesize))
+    print(utils.exec_shell_cmd('clang -g -fsanitize=fuzzer example.c -o example_fuzzer'))
+    print(utils.exec_shell_cmd('gramine-sgx-pf-crypt encrypt -w files/wrap_key -i corpus -o cipher_corpus'))
+    print(utils.exec_shell_cmd('make clean'))
+    print(utils.exec_shell_cmd('make SGX=1'))
+
+def run_libfuzzer(testname):
+    for i in range(5):
+        output = utils.exec_shell_popen('gramine-sgx libfuzz cipher_corpus', testname, 120)
```

Comment:
The timeout for each iteration, for ex: 120 in this case, have this as a Jenkins parameter. If nothing is passed then default is 120.

Reply:
added environment variable